packages/desktop-client/i18next-parser.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  input: ['src/**/*.{js,jsx,ts,tsx}', '../loot-core/src/**/*.{js,jsx,ts,tsx}'],
+  output: 'src/locale/$LOCALE.json',
+  locales: ['en'],
+  sort: true,
+  keySeparator: false,
+  namespaceSeparator: false,
+  defaultValue: (locale, ns, key, value) => {
+    if (locale === 'en') {
+      return value || key;
```

Comment:
We want to pre-fill the base `en` language file with exactly the same content as in the source.